docs/book/supported_features_matrix.md

```diff
@@ -6,22 +6,24 @@ The latest vSphere CSI Driver release is recommended for better stability and pe
+| Dynamic Block PV support (`Read-Write-Once` Access Mode)                                    | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| Dynamic Virtual Volume (vVOL) PV support                                                    | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| Topology/Availability Zone support (beta) (Block Volume only)                               | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| Static PV Provisioning                                                                      | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| K8s Multi-node Control Plane support                                                        | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| [WaitForFirstConsumer](https://kubernetes.io/docs/concepts/storage/storage-classes/) (beta) | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
+| Offline Volume Expansion support (beta) (Block Volume only)                                 | v2.0.0 to v2.3.0                          | ✅             | ✅             | ✅           | ❌             |
+| Encryption support via VMcrypt (Block Volume only)                                          | v2.0.0 to v2.3.0                          | ✅             | ✅             | ✅           | ❌             |
+| Dynamic File PV support through vSAN 7.0 File Services on vSAN Datastores                   | v2.0.0 to v2.3.0                          | ✅             | ✅             | ✅           | ❌             |
+| In-tree vSphere volume migration to CSI                                                     | v2.3.0                                    | ✅             | ✅             | ✅           | ✅             |
+| Online Volume Expansion support (beta) (Block Volume only)                                  | v2.2.0 to v2.3.0                          | ✅             | ❌             | ❌           | ❌             |
+| XFS Filesystem support (alpha)                                                              | v2.3.0                                    | ✅             | ✅             | ✅           | ✅             |
+| Raw Block Volume support (alpha)                                                            | v2.3.0                                    | ✅             | ✅             | ✅           | ✅             |
 
 _Notes_:
 
-* We do not recommend `beta` features for production use. Subsequent releases may introduce incompatible changes.
+* We do not recommend `beta` or `alpha` features for production use. Subsequent releases may introduce incompatible changes.
```

Comment:
Can we provide some details on why we are calling certain feature(s) as beta or alpha?

Reply:
How about this:

```
Alpha features are not qualified or supported by VMware Inc. They may be dropped in any future release.
Beta features are under qualification by VMware Inc. The details of the feature may be changed until it is released as GA.
```

Reply:
Added section for alpha and beta feature meaning